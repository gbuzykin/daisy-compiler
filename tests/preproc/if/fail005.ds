#if true
#else
#else
#endif

#if false
#elif
#endif

#if true
#else
#elif false
#endif