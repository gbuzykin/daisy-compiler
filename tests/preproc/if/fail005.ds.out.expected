./preproc/if/fail005.ds:3:2: error: multiple `#else` section
 3 | #else
   |  ^~~~
./preproc/if/fail005.ds:7:6: error: expected token in expression
 7 | #elif
   |      ^
./preproc/if/fail005.ds:12:2: error: `#elif` after `#else`
 12 | #elif false
    |  ^~~~
./preproc/if/fail005.ds: info: warnings 0, errors 3
